bin/main.rb

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env ruby
+require_relative '../lib/game_logic'
+require_relative '../lib/player'
+def display_board
+  puts " #{@board[0]} | #{@board[1]} | #{@board[2]} "
+  puts ' ----------- '
+  puts " #{@board[3]} | #{@board[4]} | #{@board[5]} "
+  puts ' ----------- '
+  puts " #{@board[6]} | #{@board[7]} | #{@board[8]} "
+end
```

Comment:
Our lib files are not supposed to make direct or indirect calls to puts. Instead of calling puts indirectly though this method from inside the `game_logic.rb` file, let's follow @bertil291utn's suggestion: https://github.com/rayking12/tic-tac-toe-ruby/pull/3#discussion_r535284586

Please construct the string in `game_logic.rb`, as suggested by Bertil, and then return it's value from there. This way, you can use `puts` in `main.rb` to output this value, without using `puts` (directly or indirectly) from inside `game_logic.rb`.

If you have any trouble implementing this, feel free to contact me on Slack or on any of my other social accounts!